src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -14,6 +18,8 @@ public Order completeOrder(Long userId) {
         // NOTE: In production ready code this order identifier should be generated by DB
         // For test purpose we simplify this and return dummy data
         order.setOrderId(1L);
```

Comment:
The order ID should not be hardcoded. It should be generated in a way that ensures it's unique for each order.